lib/langs.rb

```diff
@@ -1,5 +1,11 @@
 require 'ostruct'
 require 'sinatra/logger'
+require 'toml-rb'
+require 'rubygems'
+
+class Gem::Dependency
+  TYPES = [:runtime, :development, :build]
```

Comment:
Cargo files have support for a 'build-dependencies' section. However, Gem::Dependency will not accept `:build` as a valid type. I tried:
```ruby
module CuttingEdge
  class Dependency < Gem::Dependency
    TYPES = [:runtime, :development, :build]
  end
end
```
But that still doesn't work -- the `initialize` method still uses the constant as defined in the superclass. :(

This throws a warning about redefining the constant. I'm also ok with leaving out support for rust build dependencies, as the app wouldn't parse them by default anyway (only if the user sets something like `dependency_types: [:runtime, :build]` in the config)

Reply:
Something for a separate issue.